### DISCUSSION TITLE
Failed to Build Server Docker Image

### DISCUSSION THREAD

**Discussion by AlleyKilla** [metaworking/channeld-ue-plugin#108] [Q&A]
I keep getting the following error when trying to package using the Cloud Deployment tool. 

LogChanneldEditor: Error: Failed to copy ChanneldUE.ini

I can't seem to find the file anywhere. I do have a BaseChanneldUE.ini file Is there a step I might have missed that creates this file?

**Comment by indiest** [comment]
Hi! Which version of Unreal Engine are you using?
